compute/instances/reference-content/instances-datasheet.mdx

```diff
@@ -0,0 +1,175 @@
+|:--------------------------------|:------------------------------------------------------------------------------|
+| Instance Type   	              | COPARM1                                                             	        |
+| Availability Zone   	          | PAR2                                                                          |
+| Storage   	                    | Block (from 10 GB to 10 TB)                                                   |
+| Max. NIC bandwidth   	          | From 200 Mbps to 3.2 Gbps                                                     |
+| CPU Type   	                    | ARM (Ampere Altra Max M128-30)                                                |
+| CPU architecture  	            | `arm64`                                                                     	|
+| Resources   	                  | Shared vCPUs                                                                  |
+| Sizing   	                      | From 2 to 128 vCPUs <br />From 8 to 128 GiB RAM                               |
+| vCPU:RAM ratio                  | 1:4                                                                           |
+
+</Concept>
+
+<Concept>
+
+## Enterprise Instances
+
+See below the technical specifications of Enterprise Instances:
+
+| Range                         	| Production-Optimized                                                          |
+|:--------------------------------|:------------------------------------------------------------------------------|
+| Instance Type   	              | ENT1                                                                	        |
+| Availability Zone   	          | PAR1, PAR2, PAR3, AMS1, AMS2, AMS3, WAW1, WAW2, WAW3                          |
+| Storage   	                    | Block (from 10 GB to 10 TB)                                                   |
+| Max. NIC bandwidth   	          | From 400 Mbps to 20 Gbps                                                      |
+| CPU Type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |
+| CPU architecture  	            | `amd64`                                                                     	|
+| Resources   	                  | Dedicated vCPUs*                                                              |
+| Security feature                | Secure Encrypted Virtualization                                               |
+| Sizing   	                      | From 2 to 96 vCPUs  <br /> From 8 GiB to 384 GiB RAM                          |
+| vCPU:RAM ratio                  | 1:4                                                                           |
+
+\* Instances with dedicated vCPU do not share their compute resources with other Instances (1 vCPU = 1 CPU thread dedicated to that Instance). This type of Instance is particularly recommended for running production-grade compute-intensive applications
```

Comment:
Do we add a full stop at the end of this sentence? (same for the one below)

Reply:
```suggestion
\* Instances with dedicated vCPU do not share their compute resources with other Instances (1 vCPU = 1 CPU thread dedicated to that Instance). This type of Instance is particularly recommended for running production-grade compute-intensive applications.
```